theme/static/css/hydroshare_core.css

```diff
@@ -4863,4 +4863,12 @@ div.property-selector {
 .word-break-normal {
   word-break: normal;
   overflow-wrap: anywhere;
+}
+
+.tr-download-button{
```

Comment:
instead of adding to the end of the file, perhaps move this up near the other `.custom-btn-toolbar` properties